manage/config-dialog.js

```diff
@@ -197,6 +197,10 @@ function configDialog(style) {
     renderValues();
   }
 
+  function isNumber(value) {
```

Comment:
Since this would detect numeric string, maybe it should be renamed to `isNumeric`.

Reply:
I didn't make this change, but instead changed the function - see comment below.